src/graphviz.rs

```diff
@@ -34,3 +92,21 @@ pub fn write(output: &mut dyn io::Write, root: &SyntaxNode) -> FormatterResult<(
 
     Ok(())
 }
+
+#[cfg(test)]
+mod test {
+    use super::escape;
+
+    #[test]
+    fn double_escape() {
+        // PBT would be handy, here
```

Comment:
Better expand that TLA.

Reply:
Property based testing, à la [`quickcheck`](https://crates.io/crates/quickcheck) or [`proptest`](https://crates.io/crates/proptest).

Reply:
Yeah, I meant in the code comment 🙂